Dockerfile

```diff
@@ -21,8 +21,6 @@ ENV JUPYTER_ENABLE_LAB=yes
 # I don't see why we need a work subdirectory in home
 RUN rm -rf ${HOME}/work
 
-# Copy the artifacts from stage 0 that we need
-COPY --from=builder /opt/conda /opt/conda
 
 # Ensure that the conda provided PROJ database is found. This
```

Comment:
Unfortunately the wrong statement was deleted. We do not need a PROJ database. But we do need the conda environment from the build stage. So previous line 25 needs to be restored and previous line 30 needs to go.